src/integration/pages/idxPage/canonicalTests.js

```diff
@@ -0,0 +1,47 @@
+import {
+  runCoreCanonicalTests,
+  runCanonicalAnalyticsTests,
+} from '../../common';
+
+export default () => {
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+
+  it('I can see a radio schedule component with an id', () => {
+    const hasRadioSchedule = service === 'persian';
+    const id = document.getElementById('Radio-Schedule');
+
+    if (hasRadioSchedule) {
+      expect(id).toBeInTheDocument();
+    } else {
+      expect(id).not.toBeInTheDocument();
+    }
+  });
+
+  describe('Story Promo', () => {
```

Comment:
Is it worth moving the Story Promo tests to crossPlatform?

The main difference I could see was the image selector (`img` vs `amp-img`), and you might be able to combine them

Reply:
Yeah, thanks for that suggestion @karinathomasbbc .